### DISCUSSION TITLE
Unnecessary Spacing Token

### DISCUSSION THREAD

**Discussion by emersonlaurentino** [vtex/shoreline#1604] [General]
Why do we need the `$space-gap` if we already have the `$space-4`?

It's like a proxy/rename but without benefits. In reality, this is another evil, as it creates another, unnecessary token.

[Start of this discussion](https://github.com/vtex/shoreline/pull/1593#discussion_r1579817781)

**Comment by davicostalf** [comment]
@matheusps can you comment on this?
